pyproject.toml

```diff
@@ -29,6 +35,10 @@ csvcubed-devtools = "0.1.8"
 requests-mock = "~1.9.3"
 pytest-vcr = "~1.0.2"
 pytest-timeout = "^2.1.0"
+scalene = "^1.5.14"
```

Comment:
Can you remove scalene since we've decided not to use it?

Reply:
yeah, done